src/main/java/io/functionmesh/compute/rest/api/MeshComponentImpl.java

```
@@ -163,8 +163,12 @@ public void deregisterFunction(final String tenant,
             if (!StringUtils.isEmpty(authPluginName)) {
                 AuthHandler handler = CommonUtil.AUTH_HANDLERS.get(authPluginName);
                 if (handler != null) {
-                    handler.cleanUp(worker(), clientRole, clientAuthenticationDataHttps, apiKind, clusterName, tenant,
-                            namespace, componentName);
+                    try {
+                        handler.cleanUp(worker(), clientRole, clientAuthenticationDataHttps, apiKind, clusterName, tenant,
+                                namespace, componentName);
+                    } catch (RuntimeException e) {
+                        log.error("clean up auth for {}/{}/{} {} failed", tenant, namespace, componentName, e.getMessage());
```

Did you intend to eat the exception here?  Not being logged very well either.